web/README.md

```diff
@@ -23,6 +23,98 @@ $ npm start
 This command starts a local development server and opens up a browser window.
 Most changes are reflected live without having to restart the server.
 
+### Writing docs
```

Comment:
Hm this pulled in the changes from Carlos's PR, that is kind of wrong! We don't want that in this PR really.
I think you branched from main and then made a PR toward release? You should instead branch from release and then make a PR toward release.